discordrb.json

```diff
@@ -57,6 +57,18 @@
             "voiceReceive": "no",
             "voiceSend": "yes",
             "voiceStateUpdate": "yes"
+        },
+        "misc": {
+            "supportsUserLogin": "yes",
+            "supportsBotLogin": "yes",
+            "cachesTokens": "yes",
+            "apiDocs": "inline",
+            "tutorials": "yes",
```

Comment:
What I mean is:
- Tutorials: pages in a documentation somewhere that explain how to use some feature (will likely include short example snippets)
- Examples: full runnable scripts that also demonstrate a particular feature